docs/reference/errors-and-warnings/NU3003.md

```diff
@@ -40,4 +40,8 @@ NuGet client tried to verify a package which contains an invalid package signatu
 
 Please file an issue at [NuGet/Home](https://github.com/NuGet/Home/issues) along with the package that generated this problem and the platform on which the issue was encountered.
 
+## Note
+
+When NuGet’s [signature validation mode] (https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages#configure-package-signature-requirements) is set to accept (default), NU3003 is raised as a warning.
```

Comment:
Remove whitespace between `]` and `(`.

    [signature validation mode](https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages#configure-package-signature-requirements)

The whitespace screws up the hyperlinking.

    [incorrect] (url)
    [correct](url)

This feedback applies to all similar sentences in this PR, not just this one for NU3003.

Reply:
Thanks! Fixed in NU3003, NU3009,NU3010,NU3011,NU3012,NU3013,NU3014,NU3015,NU3016,NU3017,NU3018,NU3028,NU3037.